controllers/disruption_controller.go

```diff
@@ -131,9 +131,9 @@ func (r *DisruptionReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	// handle any chaos pods being deleted (either by the disruption deletion or by an external event)
 	if err := r.handleChaosPodsTermination(instance); err != nil {
 		if isModifiedError(err) {
-			r.log.Warnw("error handling chaos pods termination", "error", err)
+			r.log.Infow("retryable error handling chaos pods termination", "error", err)
 
-			return ctrl.Result{}, nil
+			return ctrl.Result{Requeue: true}, err
```

Comment:
I think the `Requeue: true` is not needed as error is not nil:
* https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/reconcile#Reconciler

```go
	// Reconcile performs a full reconciliation for the object referred to by the Request.
	// The Controller will requeue the Request to be processed again if an error is non-nil or
	// Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
```

Reply:
Yes, that is true. It's fine to be redundant, though?

Reply:
yep it is fine indeed, I only tend to use the minimal instruction to achieve expected result, only a personal matter though